src/app/components/AccountMenu/index.tsx

```diff
@@ -92,62 +91,63 @@ function AccountMenu({ showOptions = true }: Props) {
-                {tCommon("balance")}
-              </span>
-              <p className="flex justify-between">
-                <span className="dark:text-white">
-                  {accountLoading ? (
-                    <SkeletonLoader className="w-16" />
-                  ) : (
-                    balancesDecorated.accountBalance
-                  )}
-                </span>
-                <span className="text-gray-600 dark:text-neutral-400">
-                  {accountLoading ? (
-                    <SkeletonLoader className="w-12" />
-                  ) : (
-                    balancesDecorated.fiatBalance && (
-                      <>~{balancesDecorated.fiatBalance}</>
-                    )
-                  )}
-                </span>
-              </p>
-            </div>
-          </Menu.Item>
-          <Menu.ItemButton
-            onClick={() => {
-              openOptions(`accounts/${authAccount?.id}`);
-            }}
-          >
-            <PopiconsWalletLine className="w-4 h-4 mr-2 shrink-0" />
-            {tCommon("wallet_settings")}
-          </Menu.ItemButton>
-          {(isAlbyLNDHubAccount(
-            authAccount?.alias,
-            authAccount?.connectorType
-          ) ||
-            isAlbyOAuthAccount(authAccount?.connectorType)) && (
-            <Menu.ItemButton
-              onClick={() => {
-                window.open(`https://getalby.com/user`, "_blank");
-              }}
-            >
-              <PopiconsGlobeLine className="w-4 h-4 mr-2 shrink-0" />
-              {t("options.account.go_to_web_wallet")} →
-            </Menu.ItemButton>
+          {authAccount && (
+            <Menu.Item>
+              <div className="p-2 overflow-hidden">
+                <div className="flex flex-row items-center justify-between bg-amber-50 dark:bg-brand-yellow/50 border-brand-yellow border-l-4 p-2 rounded-lg">
+                  <div className="flex flex-row items-center gap-2">
+                    <div className="shrink-0">
+                      <Avatar
+                        size={24}
+                        name={authAccount.id}
+                        url={authAccount.avatarUrl}
+                      />
+                    </div>
+                    <div className="flex flex-col">
+                      <span className="overflow-hidden text-ellipsis whitespace-nowrap font-medium">
+                        {authAccount.name}
+                      </span>
+                      <span className="dark:text-white text-xs">
+                        {accountLoading ? (
+                          <SkeletonLoader className="w-16" />
+                        ) : (
+                          balancesDecorated.accountBalance
+                        )}
+                      </span>
+                    </div>
+                  </div>
+                  <div className="flex flex-row gap-1 items-center">
+                    {(isAlbyLNDHubAccount(
```

Comment:
is it always true that albylndhub account users can manage their account on getalby.com. for eg. old lndhub users who just used signup form in extension we had before and got started with it.

Reply:
i will merge this pr for now